hotsos/core/checks.py

```diff
@@ -766,3 +766,19 @@ def core(self):
         # go fetch
         self.all
         return self._core_snaps
+
+
+class SSLCertificatesChecksBase(object):
+
+    def __init__(self, certificate):
+        """
+        @param certificate: Expects an SSLCertificate object.
+        """
+        self.certificate = certificate
+
+    @property
+    def certificate_expires_soon(self):
+        "returns True if certificate expires in less than 60 days"
+        if self.certificate.certificate_days_to_expire <= 60:
```

Comment:
I think i'd prefer to have the expiry (60) as a constant/property and have the scenario call it from format-dict rather than statically set in the comment. that way if ever it changes we only have to change in one place.